.github/workflows/test-deploy-e2e.yaml

```diff
@@ -4,21 +4,25 @@ on:
   workflow_dispatch:
     inputs:
       environment:
-        description: 'Environment to run e2e tests against'
+        description: 'Target environment'
         type: choice
         required: true
-        default: 'dev-preprod'
         options:
-          - live-preprod
-          - live-preview
-          - dev-preprod
-          - dev-preview
-          - staging-preprod
+          - 'dev'
+          - 'staging'
+          - 'live'
```

Comment:
maybe we can have field for url instead?

Reply:
Based on my experience this is the more convenient way to give this input from github actions: some choices must be taken based on the networks, some other from the environment.

Reply:
More than this, maintaining the three dimensions (network, environment, cluster) is much more easy than maintaining each element of the resulting matrix of possible URLs.

Reply:
I was thinking more of a string field for URL, maybe an optional field that takes preference over this matrix?

Use case: when we temporarily deploy something under a different domain

we can leave it as it is for the time being until it causes us problems

Reply:
Nice suggestion! Thank you. I'm working to make this section of input standard and shared between all the test github actions. I agree leaving this as is, since this is not an urgent requirement, but I'll implement it when I'll do the centralized input.
Thank you